src/pages/patientView/patientHeader/PatientHeader.tsx

```diff
@@ -40,15 +34,15 @@ export default class PatientHeader extends React.Component<IPatientHeaderProps,
         );
     }
 
-    private getPopoverPatient(patient: typeof _ClinicalInformationData.patient) {
+    private getPopoverPatient(patient: ClinicalInformationData['patient']) {
         return patient && (
```

Comment:
TypeScript settings include strictNullChecks=true and the type of ClinicalInformationData['patient'] includes undefined. If we didn't have this check, it would crash when patient is undefined.